assets/tests/hooks/useVehiclesForBlockIds.test.ts

```diff
@@ -0,0 +1,144 @@
+import { renderHook } from "@testing-library/react-hooks"
+import useVehicleForBlockIds from "../../src/hooks/useVehiclesForBlockIds"
+import { VehicleData } from "../../src/models/vehicleData"
+import { dateFromEpochSeconds } from "../../src/util/dateTime"
+import { makeMockChannel, makeMockSocket } from "../testHelpers/socketHelpers"
+
+const vehicleData: VehicleData = {
+  id: "y1234",
```

Comment:
I see that we build `VehicleData`s at several points in the test suite as a whole, so let's make a factory for them too. Conceivably we could extract a lot of stuff out of the `Vehicle` factory and use it in both.

Reply:
Ah, yes, I had the inkling feeling I might have forgotten an opportunity to factory-ize things.

Reply:
Just pushed up some changes for this.

> Conceivably we could extract a lot of stuff out of the Vehicle factory and use it in both.
So I really _wanted_ to do this, but I couldn't really figure out how the `sequence` values would play along and how we would make sure that we could get a `Vehicle` and `VehicleData` object that we could compare like-for-like. My ultimate solution still leaves some duplication and is perhaps unsatisfying, but it's better than the way it was before.